src/Apps/Legion/Adapters/Standalone/DataManagement/ADXAdapter.ts

```diff
@@ -0,0 +1,208 @@
+                    'Content-Type': 'application/json'
+                },
+                data: {
+                    csl: '.show databases | project DatabaseName'
+                }
+            }).catch((err) => {
+                adapterMethodSandbox.pushError({
+                    type: ComponentErrorType.DataFetchFailed,
+                    isCatastrophic: true,
+                    rawError: err
+                });
+                return null;
+            });
+            return new DataManagementAdapterData<Array<string>>(
+                axiosResult?.data.Tables[0].Rows.reduce((acc, r) => {
+                    acc.push(r[0]);
+                    return acc;
+                }, [])
+            );
+        }, 'adx');
+    }
+    async createDatabase(databaseName: string) {
+        const adapterMethodSandbox = new AdapterMethodSandbox(this.authService);
+        return await adapterMethodSandbox.safelyFetchData(async (armToken) => {
+            const resourceResult = await axios({
+                method: 'post',
+                url:
+                    'https://management.azure.com/providers/Microsoft.ResourceGraph/resources',
+                headers: {
+                    Authorization: 'Bearer ' + armToken,
+                    Accept: 'application/json',
+                    'Content-Type': 'application/json'
+                },
+                params: { 'api-version': '2021-03-01' },
+                data: {
+                    query: `Resources | where type =~ 'Microsoft.Kusto/clusters' | where properties.uri =~ '${this.connectionSource}' | project id, name, location, type, tenantId, subscriptionId | order by name asc`
+                }
+            }).catch((err) => {
+                console.log(err);
+                return null;
+            });
+            const resource = resourceResult?.data?.data[0];
+            if (resource) {
+                const putDatabase = await axios({
+                    method: 'put',
+                    url: `https://management.azure.com${resource.id}/databases/${databaseName}`,
+                    headers: {
+                        Authorization: 'Bearer ' + armToken,
+                        Accept: 'application/json',
+                        'Content-Type': 'application/json'
+                    },
+                    params: { 'api-version': '2022-12-29' },
+                    data: {
+                        location: resource.location,
+                        kind: 'ReadWrite'
+                    }
+                }).catch((err) => {
+                    adapterMethodSandbox.pushError({
+                        type: ComponentErrorType.DataUploadFailed,
+                        isCatastrophic: true,
+                        rawError: err
+                    });
+                    return null;
+                });
+                return new DataManagementAdapterData(putDatabase.data);
+            }
+        }, 'azureManagement');
+    }
```

Comment:
I feel like there should be whitespace below this, kinda surprised that linting didnt add whtiespace in

Reply:
ahh push failed due to translations needed to be pulled first before I merged, so I will add this to my next PR, just fyi.